app/src/ingestion/edd/spiders/edd_spider.py

```diff
@@ -100,34 +102,46 @@ def to_markdown(self, html: str) -> str:
         )
         # Clean up markdown text: consolidate newlines; replace non-breaking spaces
         markdown = re.sub(r"\n\n+", "\n\n", markdown).replace("\u00A0", " ")
+
+        # Replace non-absolute URLs with absolute URLs. 2 scenarios:
+        parsed = urlparse(base_url)
+        domain_prefix = parsed.scheme + "://" + parsed.netloc
+        # Scenario 1: link starts with '/' like "/en/unemployment/"
+        # Prepend the domain prefix to the link
+        markdown = re.sub(r"\]\((\/.*)\)", rf"]({domain_prefix}\1", markdown)
+        # Scenario 2: link does not start with '/' or "http://" or "https://", like "unemployment/"
+        # Insert the base URL of the web page before the link
```

Comment:
Scenario 2 doesn't actually occur in the current set of EDD scraped pages but I'm including this logic for completeness. I think the CMS that EDD uses ensures that `href` links for EDD pages start with `/`.